contrib/epee/src/mlog.cpp

```diff
@@ -134,9 +135,35 @@ void mlog_configure(const std::string &filename_base, bool console, const std::s
   el::Loggers::addFlag(el::LoggingFlag::DisableApplicationAbortOnFatalLog);
   el::Loggers::addFlag(el::LoggingFlag::ColoredTerminalOutput);
   el::Loggers::addFlag(el::LoggingFlag::StrictLogFileSizeCheck);
-  el::Helpers::installPreRollOutCallback([filename_base](const char *name, size_t){
+  el::Helpers::installPreRollOutCallback([filename_base, max_log_rotation](const char *name, size_t){
     std::string rname = generate_log_filename(filename_base.c_str());
     rename(name, rname.c_str());
+    if (max_log_rotation != 0)
+    {
+      std::vector<boost::filesystem::path> found_files;
+      const boost::filesystem::directory_iterator end_itr;
+      for (boost::filesystem::directory_iterator iter(boost::filesystem::path(filename_base).parent_path()); iter != end_itr; ++iter)
+      {
+        const std::string filename = iter->path().string();
+        if (filename.size() > filename_base.size() && filename.substr(0, filename_base.size()) == filename_base)
```

Comment:
With > rather than >=, and the "if (found_files.size() > max_log_rotation)" below,  it'll keep bitmonero.log uncounted, so max-log-rotation of 1 wil keep two files.

Reply:
ok